gpii/node_modules/lifecycleManager/src/LifecycleManager.js

```diff
@@ -643,8 +643,12 @@ var gpii = fluid.registerNamespace("gpii");
         var userToken = finalPayload.userToken,
             lifecycleInstructions = finalPayload.activeConfiguration.lifecycleInstructions;
         var session = that.getSession([userToken]);
-        if (!session) {
-            fluid.fail("User with token " + userToken + " has no active session");
+        if (!session) { // if user has logged out since the update was added to queue
+            var msg = "User with token " + userToken + " has no active session, so igoring update request";
+            fluid.log(msg);
+            var promise = fluid.promise();
+            promise.reject(msg);
```

Comment:
Actually I don't think we want this to be a reject at all. It should just be a resolve - if we reject, we will blast the most likely harmless actions that are still remaining in the queue, perhaps including a pending login, etc.

Reply:
yeah, you're right.. I wasn't going back and forth between the two when writing this.. And you're right; resolving wont really do anything hurtful, unlike rejecting it might